website/content/en/docs/kubebuilder/references/client.md

```diff
@@ -0,0 +1,574 @@
+---
+title: Using the Controller Runtime Client API with Operator SDK
+linkTitle: Controller Runtime Client API
+weight: 1
+---
```

Comment:
Remove the link header. We don't want to make this public on the website just yet.
```suggestion
```